### PR TITLE
data: itsshivamsrivastava

### PR DESCRIPTION
**Pull request by itsshivamsrivastava**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6569"></a>

### CODE REVIEW

File: data/itsshivamsrivastava.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  { "icon": "FaGithub", "url": "https://github.com/itsshivamsrivastava" },
+  { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/itsshivamsrivastava/" },
+  { "icon": "FaFacebook", "url": "https://www.facebook.com/itsmeshivamsrivastava/" },
+  { "icon": "FaTwitter", "url": "https://twitter.com/shivam_142/" },
+  { "icon": "FaYoutube", "url": "https://www.youtube.com/channel/UCxLg2xlwOpmRTjeKfgayt8w" }
+  ],
+  "tags": [ 
+  "Open Source", "Javascript", "ReactJS", "NodeJS", "DevOps", "DevRel", "TypeScript", "NextJS" 
+  ],
+  "testimonials": [
```
Comment (kumarsonsoff3) [inline data/itsshivamsrivastava.json:43]: It seems like you've not yet received testimonials from these people. So it's better to remove these from your file. 🛠️

Learn more about testimonials [here.](https://linkfree.io/docs/how-to-guides/testimonials) 🙂

**Review by itsshivamsrivastava — COMMENTED** [review]
I will remove the testimonies..

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Comment by itsshivamsrivastava** [comment]
> Looks good, Thanks! 🎉
> 
> [](https://linkfree.io/docs/how-to-guides/editing)

Thanks for the marge my pull request.

**Pull request merged**